client/src/components/Home/Home.js

```diff
@@ -30,8 +30,20 @@ export const Home = () => {
   const [city, setCity] = useState('');
   const [guests, setGuests] = useState('');
   const [price, setPrice] = useState('');
+  const [priceLow, setPriceLow] = useState('');
+  const [priceUp, setPriceUp] = useState('');
   const [rating, setRating] = useState('');
 
+  const starRating4 = document.getElementById('starRating4');
```

Comment:
This is really bad react style as you interact with the dom directly. Is this really necessary? Is there no react conform way to do this?